tokens/source/color/border.json

```diff
@@ -36,8 +36,14 @@
         }
       },
       "decorative": {
-        "comment": "Border color for decorative purposes (purple.60).",
-        "value": "{color.alias.purple.60}"
+        "_": {
+          "comment": "This token has been deprecated and will be removed soon. Please use the `purple` token instead.",
+          "value": "{color.alias.purple.60}"
+        },
```

Comment:
Can we just remove this? No point in deprecating since we're still in the early stages of same comment for the other one

Reply:
I guess, yeah. Removed those (didn't use breaking change flag - I don't think we want to use it?)

Reply:
Yeah we're still in early stages so it's fine. You'll just need to check if we're using the old ones